docs/good-to-know.rst

```diff
@@ -0,0 +1,29 @@
+Good to know
+============
+
+Here we describe some important concepts related to ELFI. These will help in understanding
+how to implement custom operations (such as simulators or summaries) and can potentially
+save the user from some pitfalls.
+
+Generative model
+----------------
+
+By a generative model we mean any model that can generate some data. In ELFI the
+generative model is described with a directed acyclic graph (DAG) and the representation
+is stored in the `ElfiModel` instance. In LFI it typically includes everything from prior
```

Comment:
Perhaps in this context don't use 'LFI' or better yet define it in a separate entry. 

Could just point to Wikipedia for some words? Also for DAG.